arches/settings.py

```diff
@@ -375,6 +375,7 @@
     "corsheaders",
     "oauth2_provider",
     "django_celery_results",
+    "pgtrigger",
```

Comment:
- needs adding to template project settings file
- needs adding to upgrade instructions